### PR TITLE
Feat/352 updated sidebar

### LINKED ISSUE #352: Update design for wallet sidebar
[Eth wallet](https://www.figma.com/file/F995ESZ0QMpTBuQWGAgLRz/Vega-team-library?node-id=678%3A109377)
Screenshot 2021-10-22 at 14 32 11

[Vega wallet](https://www.figma.com/file/F995ESZ0QMpTBuQWGAgLRz/Vega-team-library?node-id=650%3A30765):
Screenshot 2021-10-22 at 14 29 31

some styling to be done in this ticket. the rest to be raised as follow up tickets after PR

### LINKED ISSUE #808: Tried to stake with the rewards but after associating a delegator everything has remained at 0
**[I have tried to repeat the fairground process and I have tried to do staking with the rewards that I have received in one day but after associating a delegator everything has remained at 0. #304](https://vega-testnet.nolt.io/304)** by Anonymous <hr/> hi, i think i found a bug. I did the fairground incentive and everything was fine but today I wanted to play with the wallet a bit and I tried to staking the rewards that have been created since yesterday. I could not since it shows me that I have 0 VEGA in associated, not associated and vega wallet. And I cant stake. 

image

0xF537735d62ce038a2027D5eFC5917F12fEB26ce6

### PR DESCRIPTION
**Pull request by dexturr**
Closes #352 
Closes #808

Needs #785 first.

Screenshot 2021-11-11 at 11 32 57

### CODE REVIEW

**Comment by campbellssource** [comment]
issue (blocking): 
The Link to connect you eth wallet is white text. it should be black.
in this screenshot I am hovering over the white text
Screenshot 2021-11-15 at 13 40 16
here without hover
Screenshot 2021-11-15 at 13 40 11

**Comment by campbellssource** [comment]
Issue (blocking):
The `__actions` div should be 100% width. or is should have the effect that each button shares approx 50% of the container.
Screenshot 2021-11-15 at 13 43 25

**Comment by campbellssource** [comment]
Request (not blocking): The VEGA in Vesting contract should not appear if sum value is 0. i.e. you do not have any tokens vesting.
Screenshot 2021-11-15 at 13 54 36

**Comment by campbellssource** [comment]
Request (non-blocking): link hover text is same as background on hover for the list of Vega keys.
hit "Change key" a list of keys appears (it should appear beneath the "change key" link but that isn't my main beef).
a list of keys appears. but the hover state is black
Screenshot 2021-11-15 at 14 09 07

**Comment by campbellssource** [comment]
question (non-blocking): @dexturr does the list of assets just show the general balance at the moment, or does it sum up all accounts in that asset?

Screenshot 2021-11-15 at 14 18 58

**Review by campbellssource — APPROVED** [review]

File: src/components/wallet-card/wallet-card.tsx [hunk 1]
```diff
@@ -62,3 +92,55 @@ export const WalletCardActions = ({
...
+export const WalletCardAsset = ({
+  image,
+  name,
+  symbol,
+  balance,
+  decimals,
+  border,
+  dark,
+}: WalletCardAssetProps) => {
+  const [integers, decimalsPlaces] = React.useMemo(() => {
```
Comment (mattrussell36) [inline src/components/wallet-card/wallet-card.tsx]: Make this a custom hook (or component) to avoid repeating this code?

**Review by mattrussell36 — CHANGES_REQUESTED** [review]
All good. Just one thing I think worth changing is to avoid repeating the integer/decimal splitting code

**Comment by dexturr** [comment]
@mattrussell36 is the intrest of throwing this to QA as fast as possible and I have addressed your comments + @campbellssource's I'm going to merge this assuming your approval 👍

**Pull request merged**